report

```diff
@@ -18,8 +18,6 @@ docker-compose pull
 
 docker-compose build
 
-exit_code="0"
```

Comment:
The reason this is done is that the exit code of the process run in the container is then forwarded to be the exit code of the script.

https://github.com/place-labs/staff-api/blob/master/test#L21-L27